wagtail/images/tests/test_image_operations.py

```diff
@@ -691,7 +796,7 @@ def test_default_quality(self):
         with patch("PIL.Image.Image.save") as save:
             fil.run(image, f)
 
-        save.assert_called_with(f, "JPEG", quality=85, optimize=True, progressive=True)
+        save.assert_called_with(f, "JPEG", quality=80, optimize=True, progressive=True)
```

Comment:
Please revert to 85. That is the default in Wagtail, and Willow (https://github.com/wagtail/Willow/blob/main/willow/plugins/pillow.py#L173)